build/external-dependencies.props

```diff
@@ -92,7 +92,7 @@
     <ExternalDependency Include="Mono.Addins" Version="$(MonoAddinsPackageVersion)" Private="true" />
     <ExternalDependency Include="MonoDevelop.Sdk" Version="$(MonoDevelopSdkPackageVersion)" Private="true" />
     <ExternalDependency Include="Moq" Version="$(MoqPackageVersion)" Private="true" />
-    <ExternalDependency Include="MsgPack.Cli" Version="$(MsgPackCliPackageVersion)" />
+    <ExternalDependency Include="MessagePack" Version="$(MessagePackPackageVersion)" />
```

Comment:
New third-party dependencies need to be approved by @Eilon. Was this already approved?

Reply:
Yep